src/OfficeHours.ts

```diff
@@ -70,43 +61,41 @@ export default class OfficeHours {
 
         if (message.content.startsWith("!question_list")) {
 
-            for (var data of this.questions) {
+            for (const data of this.questions) {
                 message.author.sendMessage(`${data.uuid}: <@${data.author}>: ${data.question}`)
             }
         }
 
         if (message.content.startsWith("!question_remove")) {
             const arr = message.content.split(" ");
 
-            if (arr.length == 2) {
-                const id = message.content.split(" ")[1];
+            if (arr.length === 2) {
+                const id = Number(message.content.split(" ")[1]);
```

Comment:
Only nitpick is that `+message.content.split(" ")[1]` is the more conventional way to convert to numbers. Otherwise, nice job 👍